src/components/ItemDrawer.tsx

```diff
@@ -0,0 +1,175 @@
+import { Drawer } from "@mui/material";
+import AddIcon from "@mui/icons-material/Add";
+import { useEffect, useState } from "react";
+import "./ItemDrawer.css";
+import {
+  DataGrid,
+  GridColDef,
+  GridRenderCellParams,
+  GridRowsProp,
+} from "@mui/x-data-grid";
+import { DataStore } from "@aws-amplify/datastore";
+import TagsList from "./TagsList";
+// import { Item } from "../models";
+
+const columns: GridColDef[] = [
```

Comment:
Here we can use the same column definition from ItemTable.tsx. You can export the columns in ItemTable.tsx and then import here to reuse them. DRY!